### ISSUE TITLE
`kli vc create` gives the error `ERR: can only join an iterable`

### ISSUE THREAD

**Issue by daidoji** [WebOfTrust/keripy#666]
## Description
When a registry hasn't been created and you try and issue a vc from the kli with `kli vc create` then you get the error `ERR: can only join an iterable`.  

## To Reproduce
1.  run `./scripts/demo/test_scripts.sh` # as a shortcut to getting some aids incepted
2. run `kli vc create --name cha1 --alias cha1 --data '[{"a": 1}]'` immediately afterwards

## Reproduction log
```bash
(keri) daidoji@LAPTOP-5KEUASIG:~/ssi/keripy$ kli vc create --name cha1 --alias cha1 --data '[{"a": 1}]'
ERR: can only join an iterable
(keri) daidoji@LAPTOP-5KEUASIG:~/ssi/keripy$ DEBUG_KLI=1 kli vc create --name cha1 --alias cha1 --data '[{"a": 1}]'
Traceback (most recent call last):
  File "/home/daidoji/ssi/keripy/src/keri/app/cli/kli.py", line 24, in main
    doers = args.handler(args)
            ^^^^^^^^^^^^^^^^^^
  File "/home/daidoji/ssi/keripy/src/keri/app/cli/commands/vc/create.py", line 20, in <lambda>
    parser.set_defaults(handler=lambda args: issueCredential(args),
                                             ^^^^^^^^^^^^^^^^^^^^^
  File "/home/daidoji/ssi/keripy/src/keri/app/cli/commands/vc/create.py", line 102, in issueCredential
    issueDoer = CredentialIssuer(name=name,
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/daidoji/ssi/keripy/src/keri/app/cli/commands/vc/create.py", line 182, in __init__
    self.creder = self.credentialer.create(regname=registryName,
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/daidoji/ssi/keripy/src/keri/vdr/credentialing.py", line 789, in create
    registry = self.rgy.registryByName(regname)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/daidoji/ssi/keripy/src/keri/vdr/credentialing.py", line 102, in registryByName
    if regrec := self.reger.regs.get(name):
                 ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/daidoji/ssi/keripy/src/keri/db/koming.py", line 282, in get
    key=self._tokey(keys))))
        ^^^^^^^^^^^^^^^^^
  File "/home/daidoji/ssi/keripy/src/keri/db/koming.py", line 92, in _tokey
    return (self.sep.join(keys).encode("utf-8"))  # iterable so join
            ^^^^^^^^^^^^^^^^^^^
TypeError: can only join an iterable
```

## Suggested fix
Return an error reflecting the fact that a registry hasn't been created yet rather than the TypeError currently returned.

**Comment by daidoji** [comment]
Actually this doesn't seem to be due to registries.  I was attempting to recreate what I think is another issue and created a registry and then got this bug when attempting to create a VC similar to in the description above.

**Comment by daidoji** [comment]
Also attempted `kli vc create --name cha1 --alias cha1 --data '{"a": 1}'` with same result.

**Comment by pfeairheller** [comment]
I have a fix for this on the way.

**Comment by pfeairheller** [comment]
Fixed in #675
